core/workflow_settings.ts

```diff
@@ -0,0 +1,46 @@
+import { dataform } from "df/protos/ts";
+
+export function readWorkflowSettings(projectDir: string): dataform.ProjectConfig {
```

Comment:
i don't think this code should be aware of `projectDir` at all. (and indeed reading `dataform.json` previously it was not!)

one of the whole reasons for the sandbox is to be able to root execution in that directory - require should (does) always start with that as root.

Reply:
Done.